util/util.go

```diff
@@ -7,13 +7,13 @@ import (
 	"strings"
 
 	"github.com/coreos/go-semver/semver"
-	"github.com/rancher/types/apis/management.cattle.io/v3"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
 )
 
 const (
 	WorkerThreads = 50
 
-	SupportedSyncToolsVersion = "0.1.22"
+	SupportedSyncToolsVersion = "0.1.25"
```

Comment:
I feel like we should use min version. Otherwise it's hard to keep up

Reply:
It's an after/before version. We don't need to update this once it's released.